string map
---
bool success
string message
